projects/kit/pipes/hint-error/hint-error-pipe.ts

```diff
@@ -0,0 +1,34 @@
+import {AsyncPipe} from '@angular/common';
+import {Inject, INJECTOR, Injector, OnDestroy, Pipe, PipeTransform} from '@angular/core';
+import {TuiFieldErrorPipe} from '@taiga-ui/kit/pipes/field-error';
+import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
+
+// @dynamic
+@Pipe({
+    name: 'tuiHintError',
+    pure: false,
+})
+export class TuiHintErrorPipe implements PipeTransform, OnDestroy {
+    private readonly asyncPipe!: AsyncPipe;
+    private readonly fieldErrorPipe!: TuiFieldErrorPipe;
+
+    constructor(@Inject(INJECTOR) readonly injector: Injector) {
+        const localInjector = Injector.create({
```

Comment:
Let's move this to private fields out of constructor. Also `injector` should be `private` and `INJECTOR` is only needed if you do not use static `Injector.create` which we do, so you might as well change it to `Injector`.